packages/fsapp/src/beta-app/development/dev-menu.component.web.tsx

```diff
@@ -78,9 +78,14 @@ const extractDevRoutes = (routes?: Routes, prefix: string= ''): string[] => {
         ...((('component' in route || 'loadComponent' in route) && route.quickDevMenu) ||
         'children' in route
           ? [
-            `${prefix}/${route.path ?? ''}`,
+            'initialPath' in route
+              ? `/${route.initialPath ?? ''}`
```

Comment:
`extractDevRoutes()` is building the routes for the Dev Menu.
As this is written it appears that a Route Collection would make a route for itself and then one for each child. This seems like there would be duplicates.

Can you check the dev menu to see if the `initialRoute` for a Route collection shows up twice?